src/uct/ib/mlx5/ib_mlx5.c

```diff
@@ -543,48 +543,108 @@ int uct_ib_mlx5_devx_uar_cmp(uct_ib_mlx5_devx_uar_t *uar,
 }
 
 #if HAVE_DEVX
-static ucs_status_t
-uct_ib_mlx5_devx_alloc_uar(uct_ib_mlx5_md_t *md, unsigned flags, int log_level,
-                           char *title, char *fallback,
-                           struct mlx5dv_devx_uar **uar_p)
+static ucs_status_t uct_ib_mlx5_devx_alloc_uar(uct_ib_mlx5_md_t *md, 
+                                               uint32_t flags,
+                                               struct mlx5dv_devx_uar **uar_p) 
 {
+    const char *pf_log_bar_size_suggestion = "Consider increasing PF_LOG_BAR_SIZE using mlxconfig tool (requires reboot)";
+    const char *allocation_type            = (flags == UCT_IB_MLX5_UAR_ALLOC_TYPE_WC) ? "WC" : "NC_DEDICATED";
```

Comment:
1. line seems too long
2. rename to uar_type_str

Reply:
done